scripts/analyze.sh

```diff
@@ -304,7 +304,23 @@ if [ $? -eq 0 ] || check_config_file ZJS_SENSOR; then
     >&2 echo Using module: Sensor
     MODULES+=" -DBUILD_MODULE_SENSOR"
     echo "export ZJS_SENSOR=y" >> zjs.conf.tmp
-    if [ $BOARD = "arduino_101" ]; then
+    if [ $DEV = "ashell" ]; then
```

Comment:
Is there a reason this configuration wouldn't work for non-ashell builds on arduino 101? i.e. could we just combine this into if [ $DEV = "ashell" ] || [ $BOARD = "arduino_101" ]